java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/OFIdleTime.java

```diff
@@ -0,0 +1,53 @@
+package org.projectfloodlight.openflow.types;
+
+import io.netty.buffer.ByteBuf;
+import org.projectfloodlight.openflow.annotations.Immutable;
+import org.projectfloodlight.openflow.exceptions.OFParseError;
+
+import com.google.common.hash.PrimitiveSink;
+import com.google.common.primitives.UnsignedInts;
+
+@Immutable
+public class OFIdleTime implements OFValueType<OFIdleTime> {
+	
+	//encoding of this time value is identical to OFDuration
```

Comment:
maybe just use `OFDuration` directly, if both a durations? (Or call it `OFTimeInterval`)?